src/igvfd/audit/file.py

```diff
@@ -138,12 +144,67 @@ def audit_tabular_file_missing_reference_files(value, system):
         'sample sort parameters',
         'tissue positions'
     ]
-    if value.get('content_type', '') not in excluded_content_types and 'reference_files' not in value:
-        detail = (
-            f'{object_type} {audit_link(path_to_text(value["@id"]), value["@id"])} '
-            f'has no `reference_files`.'
-        )
-        yield AuditFailure(audit_message.get('audit_category', ''), f'{detail} {audit_message.get("audit_description", "")}', level=audit_message.get('audit_level', ''))
+    # Refer to /igvfd/audit/file_set.py#L25 for this list as assay term IDs.
+    transcript_assay_term_names = [
```

Comment:
I think you can import this list, no need to duplicate it

Reply:
and duplicating is undesirable anyways since it requires upkeeing two lists

Reply:
Could we make this list a dictionary mapping between assay term IDs & assay term names. And those audits that need IDs may use the keys and those that need names may use values?